news/tasks/embedding.py

```
@@ -0,0 +1,76 @@
+from celery import shared_task
+from typing import List, Dict, Any
+import logging
+
+from news.services.embedding import EmbeddingService
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task(bind=True, max_retries=3)
+def create_single_embedding_task(
+    self, article: Dict[str, Any]
```

Ruff (0.14.10)

Unused function argument: `self`

(ARG001)